commit 26b4d21c9ced2b8a8d03e2b0e4413373bd0de69b
Author: Andrew Price <anprice@redhat.com>
Date:   Thu Feb 15 11:06:18 2018 +0000

    glocktop: Remove a non-existent flag from the usage string
    
    There is no -f option.
    
    Signed-off-by: Andrew Price <anprice@redhat.com>

diff --git a/gfs2/glocktop/glocktop.c b/gfs2/glocktop/glocktop.c
index bcee3d8b..cef368a0 100644
--- a/gfs2/glocktop/glocktop.c
+++ b/gfs2/glocktop/glocktop.c
@@ -1630,8 +1630,7 @@ static void parse_glocks_file(int fd, const char *fsname, int dlmwaiters,
 static void usage(void)
 {
 	printf("Usage:\n");
-	printf("glocktop [-i] [-d <delay sec>] [-f] [-n <iter>] [-sX] "
-	       "[-c] [-D] [-H] [-r] [-t]\n");
+	printf("glocktop [-i] [-d <delay sec>] [-n <iter>] [-sX] [-c] [-D] [-H] [-r] [-t]\n");
 	printf("\n");
 	printf("-i : Runs glocktop in interactive mode.\n");
 	printf("-d : delay between refreshes, in seconds (default: %d).\n", REFRESH_TIME);
